src/main/java/mafia/mafiatogether/service/dto/RoomValidateResponse.java

```diff
@@ -0,0 +1,6 @@
+package mafia.mafiatogether.service.dto;
+
+public record RoomValidateResponse(
+        boolean exist
```

Comment:
```suggestion
        Boolean exist
```
다른곳에는 primitive 값안쓰고 Integer로 했길레 boolean 값도 통일시켜주는건 어때?

Reply:
primitive 타입은 디폴트값 들어가서 wrapper가 좋은거같음